src/lightning/pytorch/trainer/call.py

```diff
@@ -53,6 +53,7 @@ def _call_and_handle_interrupt(trainer: "pl.Trainer", trainer_fn: Callable, *arg
     except BaseException as exception:
         trainer.state.status = TrainerStatus.INTERRUPTED
         trainer._call_callback_hooks("on_exception", exception)
+        trainer.strategy.on_exception(exception)
```

Comment:
Was this not added to the except block above on purpose?

Reply:
Yes, because KeyboardInterrupt is also not passed to callbacks.

Reply:
Are you referring to the `trainer._call_callback_hooks("on_exception", exception)` call? It's there too